edx_proctoring/views.py

```diff
@@ -336,27 +331,6 @@ def get(self, request, attempt_id):
             # if we never heard from the client, then we assume it is shut down
             attempt['client_has_shutdown'] = last_poll_timestamp is None
```

Comment:
Can we remove this line and `last_poll_timestamp = ...` line above (with comments)?  Can you also search and make sure `client_has_shutdown` and `last_poll_timestamp` are basically never used?